### ISSUE TITLE
[Accessibility] [Voice Over] [Select] Combo box option number out of total no. of  items is not read out in voice over

### ISSUE THREAD

**Issue by dhveerap** [microsoft/fast-dna#3066]
**Describe the bug; what happened?**
Combo box option number and total count of items is not read out in voice over.

**What are the steps to reproduce the issue?**
1) Render Select component in MAC
2) Start voice over
3) Navigate to Select component and open the menu
4) Voice over reads the focusable element but not the option number out of total number of options available

**What behavior did you expect?**
*  It should read the option number and total number of options after the focusable element
Example: one of ten, two of ten etc.

**In what environment did you see the issue?**

* OS & Device: [MacOS]
* Browser [Edge]

**Comment by chrisdholt** [comment]
Thanks @dhveerap - does this repro in other screen readers or is it specific to voiceover? Is it possible this is a VoiceOver specific issue?

**Comment by dhveerap** [comment]
@chrisdholt 

yeah, this does not repro in Narrator/NVDA.
Could be the platform bug. I tried with custom HTML and voice over does not read the number with plain <select> and <option> too.

**Comment by roryfa** [comment]
> @chrisdholt
> 
> yeah, this does not repro in Narrator/NVDA.
> Could be the platform bug. I tried with custom HTML and voice over does not read the number with plain select and option too.

Then this is a voice over bug IMO.
